### PR TITLE
Update team

### PR DESCRIPTION
**Pull request by zepfietje**
I've updated the list of team members to include @pxlrbt and me. As the list was ordered alphabetically by last name, I've kept it this way. Let me know if you want me to change the order!

### CODE REVIEW

**Comment by pxlrbt** [comment]
I think we should keep the first row as it is.

**Comment by zepfietje** [comment]
> I think we should keep the first row as it is.

That's fine by me as well. I just didn't want to introduce any bias and kept the order as it was.

**Comment by danharrin** [comment]
Gonna keep Ryanx2 & I ("founders") at the top, then everyone else underneath in alphabetical order I think

**Comment by zepfietje** [comment]
> Gonna keep Ryanx2 & I ("founders") at the top, then everyone else underneath in alphabetical order I think

Sounds good to me!

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -620,11 +620,11 @@ class="h-[8rem]"
                     </x-slot>
                 </x-testimonial>
 
-                <x-testimonial url="https://twitter.com/zepfietje/status/1486763133591072768">
-                    Need an admin panel for your Laravel project? It's <strong>powerful</strong>. It's <strong>fast</strong>. It's <strong>beautiful</strong>. It's <strong>easy</strong>.
+                <x-testimonial url="https://twitter.com/nkornel/status/1547530825117360129">
+                    We chose Filament and <strong>clients love it</strong>. It is <strong>consistent</strong> and <strong>clean</strong> so the <strong>learning curve is better</strong>.
 
-                    <x-slot name="author" avatar="https://avatars.githubusercontent.com/u/44533235?v=4">
```
Comment (zepfietje) [inline resources/views/home.blade.php:626]: Lol yeah was also thinking it's probably good to remove my testimonial now I'm part of the team, @danharrin. 😂

**Pull request merged**